src/device/arm/arm.go

```diff
@@ -56,195 +19,3 @@ func Asm(asm string)
 // You can use {} in the asm string (which expands to a register) to set the
 // return value.
 func AsmFull(asm string, regs map[string]interface{}) uintptr
-
-// Run the following system call (SVCall) with 0 arguments.
-func SVCall0(num uintptr) uintptr
```

Comment:
Why did you move all these to arm_cortex.go? At least the `svc` instruction should be implemented on basically all ARM chips. Why did you split this file in arm.go and arm_cortex.go?